src/features/users/usersSlice.ts

```diff
@@ -134,7 +134,7 @@ export const usersSlice = createSlice({
   initialState,
   reducers: {
     logoutUser: (state) => {
-      localStorage.removeItem('token');
+      api.setCookie('token', '');
```

Comment:
It would be clearer to delete the cookie, instead of setting an empty string.

Reply:
Cookie should be deleted by setting a value of `expires` to the past (ex. `Thu, 01 Jan 1970 00:00:01 GMT`).